examples/addr2line.rs

```diff
@@ -90,13 +90,24 @@ fn symbolicate<Endian>(file: &object::File, addrs: Vec<u64>)
 fn find_address<Endian>(debug_line: gimli::DebugLine<Endian>, units: &[Unit], addr: u64)
     where Endian: gimli::Endianity
 {
+    let mut current = None;
     for unit in units {
         if unit.contains_address(addr) {
             if let Ok(mut lines) = unit.lines(debug_line) {
-                if let Ok(Some((header, row))) = lines.run_to_address(&addr) {
-                    display_file(header, row);
-                    return;
-                };
+                while let Ok(Some((header, row))) = lines.next_row() {
+                    if row.address() > addr {
```

Comment:
I *think* we should be able to add the end address to each row, so we could give the row a `contains(&self, address: u64)` method. If we did this, we wouldn't need to repeatedly make owned copies in `current`.

Actually, the next row's start address isn't immediately available, and we would have to fully evaluate the next row to get that in the worst case.

Nevermind...